libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -223,7 +223,7 @@ void AC_PrecLand::init(uint16_t update_rate_hz)
 
     // create inertial history buffer
     // constrain lag parameter to be within bounds
-    _lag.set(constrain_float(_lag, 0.02f, 0.25f));
+    _lag.set(constrain_float(_lag, 0.02f, 0.25f));  // must match LAG parameter range at line 124
```

Comment:
Could you put in a static assert to enforce that instead of a comment?

Reply:
The constraint is just so that the inertial buffer doesn't become too big. We could just update the parameter description to be more verbose.. But probably doesn't hurt either to enforce

Reply:
With some people running CV based prec land which tends to be a bit more laggy, we might want to think about increasing that buffer actually (in another PR).